engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -460,37 +311,23 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
       }
   }
 
-  it should "return with excluded metadata from the metadata route" in {
-    val workflowId = WorkflowId.randomId()
-
-    val events = Seq(
-      MetadataEvent(MetadataKey(workflowId, None, "testKey1a"), MetadataValue("myValue1a")),
-      MetadataEvent(MetadataKey(workflowId, None, "testKey1b"), MetadataValue("myValue1b")),
-      MetadataEvent(MetadataKey(workflowId, None, "testKey2a"), MetadataValue("myValue2a")),
-      MetadataEvent(MetadataKey(workflowId, None, "testKey2b"), MetadataValue("myValue2b"))
-    )
-
-    publishMetadata(events)
-
-    Get(s"/workflows/$version/$workflowId/metadata?excludeKey=testKey2b&excludeKey=testKey3") ~>
-      metadataRoute ~>
-      check {
-        status should be(StatusCodes.OK)
-        val result = responseAs[JsObject]
-        result.fields.keys should contain allOf("testKey1a", "testKey1b", "testKey2a")
-        result.fields.keys should contain noneOf("testKey2b", "testKey3")
-        result.fields("testKey1a") should be(JsString("myValue1a"))
-        result.fields("testKey1b") should be(JsString("myValue1b"))
-        result.fields("testKey2a") should be(JsString("myValue2a"))
-      }
-  }
+    it should "return with excluded metadata from the metadata route" in {
+     Get(s"/workflows/$version/${MockApiService.ExistingWorkflowId}/metadata?excludeKey=testKey2b&excludeKey=testKey3") ~>
+        cromwellApiService.metadataRoute ~>
+        check {
+          status should be(StatusCodes.OK)
+          val result = responseAs[JsObject]
+          result.fields.keys should contain allOf("testKey1a", "testKey1b", "testKey2a")
+          result.fields.keys should contain noneOf("testKey2b", "testKey3")
+          result.fields("testKey1a") should be(JsString("myValue1a"))
+          result.fields("testKey1b") should be(JsString("myValue1b"))
+          result.fields("testKey2a") should be(JsString("myValue2a"))
```

Comment:
ToL I'd rather have the test actor and the test expectations get these from a single known constant source, rather than having them be magical strings in both places.